### PR TITLE
[SDK-3681] Update README with new design

### PR DESCRIPTION
**Pull request by jimmyjames**
Updates the README using the new design. Note that this does not include the new banner, as the design has not yet been finalized.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,239 +1,158 @@
 # Auth0 Java MVC Commons
 
+Easily add authentication with Auth0 to your Java Servlet applications.
```
Comment (stevehobbsdev) [inline README.md:3]: Let's find an alternative for "easily", as mentioned in Slack.

Comment (stevehobbsdev) [reply README.md:3]: The one-liner we have in the doc is:

> Auth0 SDK to add authentication to your Java Servlet applications

File: README.md [hunk 2]
```diff
@@ -1,239 +1,158 @@
...
+---
 
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2Fauth0-java-mvc-common.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2Fauth0-java-mvc-common?ref=badge_large)
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: light)" srcset="./assets/auth0_light_mode.png"   width="150">
+    <source media="(prefers-color-scheme: dark)" srcset="./assets/auth0_dark_mode.png" width="150">
+    <img alt="Auth0 Logo" src="./auth0_light_mode.png" width="150">
+  </picture>
+</p>
```
Comment (stevehobbsdev) [inline README.md:155]: We now have these URLs in the CDN, see https://auth0.slack.com/archives/C0414QJGXG9/p1666171760950799?thread_ts=1666170245.671199&cid=C0414QJGXG9

**Review by stevehobbsdev — COMMENTED** [review]
Nice, just left a couple things 👍🏻

File: README.md (hunk 1 above)
Comment (jimmyjames) [reply README.md]: lol I updated that one-liner in the doc, forgot to push the change here. Pushed a new commit to fix.

File: README.md [hunk 3]
```diff
@@ -1,239 +1,158 @@
 # Auth0 Java MVC Commons
 
+Auth0 SDK to add authentication to your Java Servlet applications.
+
 [![CircleCI](https://img.shields.io/circleci/project/github/auth0/auth0-java-mvc-common.svg?style=flat-square)](https://circleci.com/gh/auth0/auth0-java-mvc-common/tree/master)
 [![Coverage Status](https://img.shields.io/codecov/c/github/auth0/auth0-java-mvc-common.svg?style=flat-square)](https://codecov.io/github/auth0/auth0-java-mvc-common)
-[![License](http://img.shields.io/:license-mit-blue.svg?style=flat)](http://doge.mit-license.org)
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2Fauth0-java-mvc-common.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2Fauth0-java-mvc-common?ref=badge_shield)
+[![License](http://img.shields.io/:license-mit-blue.svg?style=flat)](https://doge.mit-license.org/)
+[![Maven Central](https://img.shields.io/maven-central/v/com.auth0/auth0-java-mvc-common.svg?style=flat-square)](https://mvnrepository.com/artifact/com.auth0/auth0-java-mvc-common)
```
Comment (ewanharris) [inline README.md:8]: Should this be the below, the current one renders `not found` for me
```suggestion
[![Maven Central](https://img.shields.io/maven-central/v/com.auth0/mvc-auth-commons.svg?style=flat-square)](https://mvnrepository.com/artifact/com.auth0/mvc-auth-commons)
```

File: README.md [hunk 4]
```diff
@@ -1,239 +1,158 @@
...
-* Analytics of how, when and where users are logging in.
-* Pull data from other sources and add it to the user profile, through [JavaScript rules](https://docs.auth0.com/rules).
+That's it! You have authenticated the user using Auth0.
 
-## Create a free account in Auth0
+## API Reference
 
-1. Go to [Auth0](https://auth0.com) and click Sign Up.
-2. Use Google, GitHub or Microsoft Account to login.
+- [JavaDocs](https://javadoc.io/doc/com.auth0/auth0-java-mvc-common/latest)
```
Comment (ewanharris) [inline README.md:130]: Same here, this 404s for me. Or are we updating the package name in future?

**Review by ewanharris — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (jimmyjames) [reply README.md]: Just a bad copy/paste, updated in newer commit

File: EXAMPLES.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
+# Examples using auth0-java-mvc-common
+
+- [Including additional authorization parameters](#including-additional-authorization-parameters)
+- [Organizations](#organizations)
+- [Allowing clock skew for token validation](#allow-a-clock-skew-for-token-validation)
+- [Chaning the OAuth response_type](#changing-the-oauth-response_type)
```
Comment (frederikprijck) [inline EXAMPLES.md:6]:
```suggestion
- [Changing the OAuth response_type](#changing-the-oauth-response_type)
```

File: README.md [hunk 6]
```diff
@@ -1,239 +1,156 @@
...
+
+Add the dependency via Maven:
 
 ```xml
 <dependency>
-    <groupId>com.auth0</groupId>
-    <artifactId>mvc-auth-commons</artifactId>
-    <version>1.9.2</version>
+  <groupId>com.auth0</groupId>
+  <artifactId>auth0-java-mvc-common</artifactId>
```
Comment (frederikprijck) [inline README.md:33]: Just to be sure: Is the change to `artifactId` expected?

File: README.md [hunk 7]
```diff
@@ -1,239 +1,156 @@
...
+Please do not report security vulnerabilities on the public Github issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
 
+---
 
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2Fauth0-java-mvc-common.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2Fauth0-java-mvc-common?ref=badge_large)
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: light)" srcset="https://cdn.auth0.com/website/sdks/logos/auth0_light_mode.png"   width="150">
+    <source media="(prefers-color-scheme: dark)" srcset="https://cdn.auth0.com/website/sdks/logos/auth0_dark_mode.png" width="150">
+    <img alt="Auth0 Logo" src="./auth0_light_mode.png" width="150">
```
Comment (frederikprijck) [inline README.md:151]:
```suggestion
    <img alt="Auth0 Logo" src="https://cdn.auth0.com/website/sdks/logos/auth0_light_mode.png" width="150">
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (jimmyjames) [reply README.md]: oh, good catch! No, that is not expected. New commit pushed to fix.

File: README.md (hunk 7 above)
Comment (jimmyjames) [reply README.md]: Included change in pushed commit.

File: EXAMPLES.md (hunk 5 above)
Comment (jimmyjames) [reply EXAMPLES.md]: Fixed in pushed commit

[the remark this answers is not in the document]
Comment (jimmyjames) [reply EXAMPLES.md]: Fixed in pushed commit

**Review by frederikprijck — APPROVED** [review]

**Review by ewanharris — APPROVED** [review]

**Pull request merged**